app/models/calculation_result.rb

```diff
@@ -30,30 +30,58 @@ def income_contribution
   end
 
   def gross_income
-    monetise(api_response.dig(:result_summary, :gross_income, :total_gross_income))
+    sum = if has_partner?
+            api_response.dig(:result_summary, :gross_income, :total_gross_income) +
```

Comment:
I may be missing something... Isnt this calculation being done somewhere in CFEP, i.e combining the partner and client gross_income's to give us an overall value which we apply against the thresholds (same for disposable and capital below). If so, would it not make more sense to return that value from CFEP instead of piecing it together here?

Reply:
@willc-work you're right. At the moment CFEP adds the two together for the purpose of comparing to the thresholds, but it doesn't store that sum anywhere in the DB, and what CFEP returns to EFE is basically a representation of the database. But it would make a lot more sense for CFEP to tell us the combined total to avoid having to do maths here.

I'll work up a CFEP PR.